.github/workflows/platformio.yml

```diff
@@ -0,0 +1,73 @@
+name: PlatformIO CI
```

Comment:
Have you tried pulling this version of the library as a dependency in a project since adding this file? A few libraries in the past had to remove the ini file to prevent pio processing it. Maybe that's been fixed now.

Reply:
I'll just merge it and try it later today myself.